common/Invn/InvBool.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * ________________________________________________________________________________________________________
+ * Copyright (c) 2017 InvenSense Inc. All rights reserved.
```

Comment:
Warning you need to have BSD3 license here too

Reply:
Done